src/scss/components/form/mixins/_textControl.scss

```diff
@@ -14,26 +14,27 @@
 // Efficiency: Needs Work
 @mixin __textControl($focused: false) {
   @if $focused {
-    border: 1px solid $cyan-700;
-    box-shadow: $focus-glow;
+    border-bottom: 1px solid $blue-700;
     outline: 0;
+    font-style: normal;
   }
   @else {
     @include hxPlaceholder;
 
     background-color: $gray-0;
-    border-radius: 2px;
+    border-radius: 1px;
     border: 1px solid $gray-500;
-    color: $gray-900;
+    color: $gray-800;
     font-family: inherit;
     font-size: 1rem;
+    font-style: italic;
   }
 }
 
 // Parity: OK
 // Efficiency: OK
 @mixin __textControl--invalid($focused: false) {
-  border: 2px solid $red-900;
+  border-bottom: 2px solid $red-status-500;
```

Comment:
Please change from `2px` to  `1px` thickness.

Reply:
Changes have been done, please review.